client/components/Button/Button.css

```diff
@@ -13,17 +13,4 @@
     margin-left: calc($base-spacing * 2);
     box-shadow: $button-shadow;
   }
-
-  &.homepageClickGrey {
```

Comment:
Hm I think you accidentally committed debugging code to your master? maybe it's better to start a clean master again

Reply:
Noted.. thank you